js/entity_browser.multi_step_display.js

```diff
@@ -20,9 +27,10 @@
       var $toggle = $('.entity-browser-show-selection');
 
       function setToggleText() {
-        if($entities.css('display') == 'none') {
+        if ($entities.css('display') == 'none') {
```

Comment:
for better readability of the changeset its IMHO better to not include those unrelated changes, but rather add them to its own PR